_data/hosting_providers.json

```diff
@@ -2208,5 +2208,15 @@
     "plan": "",
     "reviewed": "2019.7.22",
     "note": ""
-  }
+  },
+  {
```

Comment:
Order matters; you'll want to add this entry alphabetized in the right section.

Reply:
this note was resolved but I'm still seeing it at the end of the list; is there a commit that needs to be pushed?